music21/romanText/tsvConverter.py

```diff
@@ -717,18 +779,26 @@ def _m21ToTsv_v1(self):
             thisEntry.measure = thisRN.measureNumber
             thisEntry.beat = thisRN.beat
             thisEntry.totbeat = None
-            thisEntry.timesig = thisRN.getContextByClass(meter.TimeSignature).ratioString
-            thisEntry.op = self.m21Stream.metadata.opusNumber
-            thisEntry.no = self.m21Stream.metadata.number
-            thisEntry.mov = self.m21Stream.metadata.movementNumber
+            ts = thisRN.getContextByClass(meter.TimeSignature)
+            if ts is None:
+                thisEntry.timesig = ''
+            else:
+                thisEntry.timesig = ts.ratioString
+            thisEntry.extra["op"] = self.m21Stream.metadata.opusNumber
+            thisEntry.extra["no"] = self.m21Stream.metadata.number
+            thisEntry.extra["mov"] = self.m21Stream.metadata.movementNumber
             thisEntry.length = thisRN.quarterLength
             thisEntry.global_key = global_key
             thisEntry.local_key = thisRN.key.tonicPitchNameWithCase
             thisEntry.pedal = None
             thisEntry.numeral = thisRN.romanNumeral
             thisEntry.form = getForm(thisRN)
             # Strip any leading non-digits from figbass (e.g., M43 -> 43)
-            thisEntry.figbass = re.match(r'^\D*(\d.*|)', thisRN.figuresWritten).group(1)
+            figbassm = re.match(r'^\D*(\d.*|)', thisRN.figuresWritten)
+            if figbassm is not None:
+                thisEntry.figbass = figbassm.group(1)
```

Comment:
write as figbassMatch or something like that.  It looks like it means small figured bass.